UnicornRender/video/source/Mesh.cpp

```diff
@@ -43,5 +45,22 @@ const Material& Mesh::GetMaterial() const
 {
     return m_material;
 }
+
+void Mesh::Scale(glm::vec3 scale)
+{
+    m_scale = scale;
+}
+
+void Mesh::Calculate()
+{
+    CalculateOrientation();
+
+    auto T = glm::translate(glm::mat4(1.0), m_translation);
+    auto R = mat4_cast(m_orientation) * glm::mat4(1.0);
+    auto S = scale(glm::mat4(1.0), { m_scale });
```

Comment:
I don't think that this is a good idea, at this point I can only make the difference between local and glm function calls just because we have a convention naming function starting with the capital letter. Half of the code uses `glm::` and the other half does not.